About.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles1.css">
+    <title>Mobile World Congress 2023</title>
+</head>
+<body>
+    <header>
+        <div class="topbar">
+            <a href=""><i class="fa fa-facebook"></i></a>
+            <a href=""><i class="fa fa-twitter"></i></a>
+            <a href="">French</a>
+            <a href="">My Page</a>
+            <a href="">Barcelona</a>
+        </div>
+        <img src="./img/hamburger.svg" class="menu" alt="menu">
+
+        <nav class ="Desktop-navbar">
+             
+            <a href="./index.html"><img src="./img/MWClogo.png" alt="MWC" class="mylogo" /></a>
+            
+            <ul class="desktop-menu">
+
+                <li class="about"><a class="about" href="./About.html">About</a></li>
```

Comment:
- [ ] [optional] @Hisoka37 great work on your project, since you are already on the about page, it doesn't make sense having link to about  page, would recommend you change it to homepage link.